chapters/ch04.md

```diff
@@ -2,31 +2,49 @@
 
 ->
 
+# Quy Mô Quyết Định Quá Trình Machine Learning
+
 
 Many of the ideas of deep learning (neural networks) have been around for decades. Why are these ideas taking off now?
 
 ->
 
+Rất nhiều những ý tưởng của học sâu (mạng neural) đã xuất hiện từ hàng chục năm trước.  Vậy tại sao tới tận bây giờ chúng mới cất cánh, mới bùng nổ như vậy?  
+
+
+
 
 Two of the biggest drivers of recent progress have been:
 
 ->
 
+Hai nguyên nhân mang tính quyết định tới những phát triển gần đây là
+
 
 * **Data availability.​** People are now spending more time on digital devices (laptops, mobile devices). Their digital activities generate huge amounts of data that we can feed to our learning algorithms.
 
 ->
 
+**Dữ liệu sẵn có**. Ngày nay,  mọi người dành nhiều thời gian hơn bên những thiết bị số như máy tính xách tay, điện thoại di động, .v.v. Và chính những thiết bị số này là nguồn tạo ra lượng dứ liệu cực lớn - dữ liệu mà chúng ta vẫn dùng cho những thuật toán học máy.  
+
+
+
 
 * **Computational scale.** ​We started just a few years ago to be able to train neural networks that are big enough to take advantage of the huge datasets we now have.
 
-->
+-> 
+
+**Quy mô năng lực tính toán**. Chỉ vài năm gần đây chúng ta mới bắt đầu huấn luyện mạng neural đủ lớn để tận dụng lượng dữ liệu lớn kể trên. 
+
+ 
 
 
 In detail, even as you accumulate more data, usually the performance of older learning algorithms, such as logistic regression, “plateaus.” This means its learning curve “flattens out,” and the algorithm stops improving even as you give it more data:
 
 ->
 
+Cho dù có thêm nhiều nhiều dữ liệu nữa, thì với những thuật toán học máy thế hệ cũ như hồi quy logistic cũng chẳng mảy may suy chuyển. Nghĩa là đồ thị chất lượng học nằm thẳng tưng, và thuật toán sẽ không cải thiện chất lượng cho dù có cho thêm nhiều dữ liệu đi nữa. 
```

Comment:
- " thì với những thuật toán học máy" -> "thì hiệu quả của những thuật toán học"